.github/workflows/testing-alpine.yml

```diff
@@ -0,0 +1,35 @@
+name: build and test on AlpineLinux
+
+on:
+  push:
+    branches: [ master, devel ]
+  pull_request:
+    branches: [ master, devel ]
+
+jobs:
+  testing:
+    strategy:
+      fail-fast: false
+      matrix:
+        tag: ['3.20','3.19','3.18','3.17']
+        build-type: [Debug,Release]
+
+    runs-on: ubuntu-latest
+
+    container: alpine:${{ matrix.tag }}
+
+    steps:
+      - run: apk add git bash libc++-dev g++ cmake ninja
+
+      - uses: actions/checkout@v4
+        with:
+          submodules: recursive
+
+      - run: cmake -S . -B build.d -G Ninja -DCMAKE_BUILD_TYPE=${{ matrix.build-type }} -DCMAKE_VERBOSE_MAKEFILE=ON -DCMAKE_COLOR_MAKEFILE=ON
+      - run: cmake --build build.d
+
+      - run: readelf -h build.d/pegof
+      - run: readelf -d build.d/pegof
+
+      - run: build.d/pegof --help
+      - run: build.d/pegof --version
```

Comment:
This doesn't hurt, but I'd prefer to actually run the tests to check if the code is really working :slightly_smiling_face:

Reply:
On musl-based Linux system, tests can not be run due to lack of sanitizer.

Reply:
Tests should work now, but they'll require few more packages to be installed:
```
apk --no-cache add git bash libc++-dev g++ cmake ninja bats diffutils
```